test/testDistance.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env mocha */
+import {formatDistance} from '../src/formatDistance'
+
+var assert = require('chai').assert
```

Comment:
again, quick convention thing - all of our tests in rwgps-ui are written using BDD syntax (`chai.expect`). Would recommend using that here unless there's a specific reason not to.